public_html/wp-content/mu-plugins/blocks/source/blocks/schedule/front-end.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * Internal dependencies
+ */
+import { getDerivedSessions } from './data';
+import { ScheduleGridContext } from './edit';
+import { ScheduleGrid } from './schedule-grid';
+import renderFrontend from '../../utils/render-frontend';
+import './front-end.scss';
+
+// See `pass_global_data_to_front_end()` for details on front- vs back-end data sourcing.
+const rawScheduleData = window.WordCampBlocks.schedule || {};
+
+/*
+ * todo
+ *
+ * this file is being loading in the editor, but should only load on the front end.
+ * probably a similar problem mentioned in init(), so maybe conditionally register it if on the front end?
+ * `wp_using_themes() && ! wcorg_is_rest_api_request()` ? - https://wordpress.stackexchange.com/a/360401/3898
+ *
+ * the front-end build file is really big
+ * still seeing a bunch of dependencies being added to it, like classnames, lodash, memoize, @babel and @emotion,
+ * etc.
+ * also contains stuff from other custom blocks, like components/image/avatar.js, even though not used in this
+ * block.
```

Comment:
That's from importing from the main `/components` export — this is generally fine in the editor, since we usually want all those at some point, but for the front-end file you'll want to be explicit about imports (pull in directly from the component file).

Reply:
👍 951a0a0